### PR TITLE
Asian Factions Update: KPA Boogaloo

### PR DESCRIPTION
**Pull request by CSLeader**
A new update for the FPARMA repo that provides more premade assets of North Korea, China, and Japan for missionmakers.

Check the commits for a more detailed changelog.

### CODE REVIEW

File: FP_Factions/JSDF/config.cpp [hunk 1]
```diff
@@ -147,16 +521,881 @@ class cfgvehicles
...
+		magazines[] = {"rhsusf_50Rnd_762x51","rhsusf_50Rnd_762x51","rhsusf_50Rnd_762x51","rhsusf_50Rnd_762x51","RH_15Rnd_9x19_SIG","RH_15Rnd_9x19_SIG","RH_15Rnd_9x19_SIG","RH_15Rnd_9x19_SIG","HandGrenade","HandGrenade","SmokeShell","SmokeShellRed"};
+		respawnMagazines[] = {"rhsusf_50Rnd_762x51","rhsusf_50Rnd_762x51","rhsusf_50Rnd_762x51","rhsusf_50Rnd_762x51","RH_15Rnd_9x19_SIG","RH_15Rnd_9x19_SIG","RH_15Rnd_9x19_SIG","RH_15Rnd_9x19_SIG","HandGrenade","HandGrenade","SmokeShell","SmokeShellRed"};
+	};
+};
+
+class cfgWeapons
+{
+	class rhs_weap_m4a1_blockII_bk;
+	class rhs_weap_m4a1_blockII_M203_bk;
+	class rhs_weap_m4a1_blockII_bk_nt4_black_compm4_rvg_blk: rhs_weap_m4a1_blockII_bk
```
Comment (diwako) [inline FP_Factions/JSDF/config.cpp:1301]: when adding new guns, please use the `FP_` prefix so it is clear this addition is made by use and not rhs, hlc etc

File: FP_Factions/KPA/config.cpp [hunk 2]
```diff
@@ -373,294 +528,2421 @@ class CfgGroups
 						position[] = {5,-5,0};
 					};
 				};
+
 			};
+<<<<<<< HEAD
```
Comment (diwako) [inline FP_Factions/KPA/config.cpp:533]: oh?

File: FP_Factions/KPA/config.cpp [hunk 3]
```diff
@@ -373,294 +528,2421 @@ class CfgGroups
...
-		crew = "FP_Faction_KPA_Rifleman";
-		faction = "FP_Faction_KPA";
-	};	
-	class UK3CB_O_G_T55;
-	class FP_Faction_KPA_Chonmaho: UK3CB_O_G_T55
-	{
-		class TextureSources
-		{
-			class KPA_T55_Green
+=======
```
Comment (diwako) [inline FP_Factions/KPA/config.cpp:538]: oh??

File: FP_Factions/KPA/config.cpp [hunk 4]
```diff
@@ -373,294 +528,2421 @@ class CfgGroups
...
+						rank = "CORPORAL";
+						position[] = {5,-5,0};
+					};
+				};
+			};
+		};
+	};
+};
+class CfgVehicles
+>>>>>>> 399ef40... Factions Update Ver. I forgot at this point
```
Comment (diwako) [inline FP_Factions/KPA/config.cpp:1368]: ooooooooh?

File: FP_Factions/KPA/config.cpp [hunk 5]
```diff
@@ -373,294 +528,2421 @@ class CfgGroups
...
+	};
+<<<<<<< HEAD
+	class UK3CB_O_G_BTR40;
+	class UK3CB_O_G_BTR40_MG; 
+	class FP_Faction_KPA_BTR40: UK3CB_O_G_BTR40
+=======
+	class UK3CB_CW_SOV_O_EARLY_BTR40;
+	class UK3CB_CW_SOV_O_EARLY_BTR40_MG; 
+	class FP_Faction_KPA_BTR40: UK3CB_CW_SOV_O_EARLY_BTR40
+>>>>>>> 399ef40... Factions Update Ver. I forgot at this point
```
Comment (diwako) [inline FP_Factions/KPA/config.cpp:1384]: oh nooooooooooo

File: FP_Factions/KPA/config.cpp [hunk 6]
```diff
@@ -373,294 +528,2421 @@ class CfgGroups
...
+			};
+		};
+		author = "FP Mod Team";
+		crew = "FP_Faction_KPA_Crewman";
+		displayName = "Ch'ŏnma-ho";
+		faction = "FP_Faction_KPA";
+		hiddenSelections[] = {"camo1","camo2"};
+		hiddenSelectionsTextures[] = {"\fp_factions\KPA\data\t55\t55_body_kpa_co.paa","\fp_factions\KPA\data\t55\t55_tower_kpa_co.paa"};
+	};
+<<<<<<< HEAD
```
Comment (diwako) [inline FP_Factions/KPA/config.cpp:1418]: oooooooo

File: FP_Factions/KPA/config.cpp [hunk 7]
```diff
@@ -373,294 +528,2421 @@ class CfgGroups
...
+		crew = "FP_Faction_KPA_Rifleman";
+		faction = "FP_Faction_KPA";
+	};	
+	class UK3CB_CW_SOV_O_EARLY_ZsuTank;
+	class FP_Faction_KPA_ZSU_234_AA: UK3CB_CW_SOV_O_EARLY_ZsuTank
+	{
+		crew = "FP_Faction_KPA_Crewman";
+		faction = "FP_Faction_KPA";
+	};
+>>>>>>> 399ef40... Factions Update Ver. I forgot at this point
```
Comment (diwako) [inline FP_Factions/KPA/config.cpp:1477]: ohoh

File: FP_Factions/KPA/config.cpp [hunk 8]
```diff
@@ -373,294 +528,2421 @@ class CfgGroups
...
+	class FP_Faction_KPA_UAZ: rhsgref_nat_uaz
+	{
+		side = 0;
+		crew = "FP_Faction_KPA_Rifleman";
+		editorSubcategory = "EdSubcat_Cars";
+		faction = "FP_Faction_KPA";
+	};
+	class FP_Faction_KPA_UAZ_Open: rhsgref_nat_uaz_open
+	{
+<<<<<<< HEAD
```
Comment (diwako) [inline FP_Factions/KPA/config.cpp:1492]: ooooohhhhhhh

**Comment by CSLeader** [comment]
I found more errors in this PR such as duplicate classnames. I'll make another commit to fix this.

**Comment by CSLeader** [comment]
Alright, this should fix everything mentioned above

**Pull request merged**